cmd/operator/app/command.go

```diff
@@ -90,6 +91,11 @@ func NewOperatorCommand() *cobra.Command {
 				os.Exit(1)
 			}
 
+			if err := greptimedbstandalone.Setup(mgr, o); err != nil {
+				setupLog.Error(err, "unable to setup controller", "controller", "greptimedbstandalone")
+				os.Exit(1)
+			}
+
```

Comment:
suggestion：Use greptimedb standalone need to install crd, default not setup?

Reply:
same as `greptimedbcluster`. you need to `make install` to install greptimedbstandalone.